test/ctl/ctl_debug.h

```diff
@@ -1,20 +1,20 @@
 /*
  *
- * Copyright (C) 2024 Intel Corporation
+ * Copyright (C) 2024-2025 Intel Corporation
  *
  * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  *
  */
 
 /*
- * ctl_debug.h -- definitions for CTL test
+ * ctl_debug.h -- definitions for CTL tests
  */
 
-#ifndef UMF_CTL_DEBUG_H
-#define UMF_CTL_DEBUG_H 1
+#ifndef UMF_TEST_CTL_DEBUG_H
+#define UMF_TEST_CTL_DEBUG_H 1
 
-#include "../src/ctl/ctl.h"
+#include "../src/ctl/ctl_internal.h"
```

Comment:
could you try just "ctl/ctl_internal.h" ? src should be added to search paths. If not - please add it

Reply:
done